components/brave_rewards/browser/rewards_service.h

```diff
@@ -244,10 +244,10 @@ class RewardsService : public KeyedService {
 
   virtual void RemoveRecurringTip(const std::string& publisher_key) = 0;
 
-  virtual void SetMonthlyContribution(
```

Comment:
Wild question - I'll have to check, but isn't this an unused API? I thought we handle tips through `RewardsServiceImpl::OnTip()`, and then `bat_ledger_->OneTimeTip()` and `bat_ledger_->SaveRecurringTip()`.

Reply:
Yes, I should have made that more clear in the PR description. This is a new API for the new tipping banner.